### PR TITLE
Update ns-wtsapi32-wts_client_address.md

### PR DESCRIPTION
**Pull request by asklar**
Fix Address comment for AF_INET

### CODE REVIEW

**Comment by PRMerger7** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sdk-api-src/content/wtsapi32/ns-wtsapi32-wts_client_address.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Address family. This member can be <b>AF_INET</b>, <b>AF_INET6</b>, <b>AF_IPX</b
 
 Client network address. The format of the field of <b>Address</b> depends on the address type as specified by the <b>AddressFamily</b> member.
 
-For an address family <b>AF_INET</b>: <b>Address </b> contains the IPV4 address of the client as a null-terminated string.
+For an address family <b>AF_INET</b>: <b>Address </b> contains the IPV4 address of the client as raw byte values. For example, the address 192.168.0.1 would be represented as the following series of byte values: "0x00 0x00 0xC0 0xA8 0x00 0x01".
```
Comment (asklar) [inline sdk-api-src/content/wtsapi32/ns-wtsapi32-wts_client_address.md]:
```suggestion
For an address family **AF_INET**: **Address** contains the IPV4 address of the client as raw byte values. The IP address is offset by two bytes from the start of the Address member. For example, the address 192.168.0.1 would be represented as the following series of byte values: "0x00 0x00 0xC0 0xA8 0x00 0x01".
```

**Comment by PRMerger-2** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by QuinnRadich** [comment]
#sign-off

**Pull request merged**